source/js/embed-tf.js

```diff
@@ -0,0 +1,29 @@
+let EmbedTF = {
+  init: function() {
+    document.addEventListener("DOMContentLoaded", () => EmbedTF.setup());
```

Comment:
This will not fire if `DOMContentLoaded` has already happened. As per https://developer.mozilla.org/en-US/docs/Web/API/Document/DOMContentLoaded_event#Checking_whether_loading_is_already_complete, let's change this to:
```suggestion
    if (document.readyState === 'loading') {
      document.addEventListener('DOMContentLoaded', () => EmbedTF.setup());
    } else {
      EmbedTF.setup();
    }
```

Reply:
Nice catch!